### DISCUSSION TITLE
Publishing project API Docs

### DISCUSSION THREAD

**Discussion by stroudcuster** [community/community#38422] [New to GitHub] [answered]
How can I publish my API docs (HTML, generated by Sphinx) on GitHub?

**Comment by stroudcuster** [comment, marked as answer]
Thanks.  My API docs have been successfully published.  Thanks for the
article. This is my first experience with Sphinx, and I've found it
difficult to get the markup to work as advertised, so any info is welcome.


On Nov 6, 2022 at 11:10 AM, David Engelmann ***@***.***>
wrote:

You can use Github pages, check out this write up on it
<https://www.docslikecode.com/articles/github-pages-python-sphinx/>

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/community/discussions/38422#discussioncomment-4069415>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AZHNUOEAFR3JTCDYUMQO2BTWG7C7HANCNFSM6AAAAAARYPTF2E>
.
You are receiving this because you authored the thread.Message ID:
***@***.***>

**Reply by david-engelmann** [reply]
Glad to hear it! Make sure to make "Mark as Answer"!

**Reply by stroudcuster** [reply]
Done.  Thanks again.  I tried publishing to ReadTheDocs and for some reason it could resolve any of my modules when it tried to build my app.  I'm glad GitHub had a 'publish static docs' path since I already had all the docs built.

**Reply by david-engelmann** [reply]
Usually people don't mark themselves as the answer :)

**Reply by david-engelmann** [reply]
Just realized you can select comments

**Comment by david-engelmann** [comment]
[You can use Github pages, check out this write up on it](https://www.docslikecode.com/articles/github-pages-python-sphinx/)
